packages/plugin-workflow-approval/src/client/user-interface/h5/todos/component/TabApprovalItem.tsx

```
@@ -86,11 +88,11 @@ const approvalTodoListStatus = (item, t) => {
   const { workflow, execution, job, status } = item;
   if (
     (!(workflow != null && workflow.enabled) || (execution != null && execution.stauts) || job?.status) &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)